docs/crds/hostrule.md

```diff
@@ -141,6 +146,18 @@ DNS will arrive with the host header as foo.com to the VIP hosting foo.region1.c
 
 This knob is currently only supported with the SNI model and not with Enhanced Virtual Hosting model.
 
+#### Configure Analytics Policy
+
+The HostRule CRD can be used to configure analytics policies such as enable/disable non-significant logs, throttle the number of non-significant logs per second on each SE, enable/disable logging of all headers, etc.
+
+        analyticsPolicy:
+          fullClientLogs:
+            enabled: true
+            throttle: HIGH
```

Comment:
could you also please elaborate on the enums and their usage here.

Reply:
I have addressed the comment.